rust/rsc/src/rsc/main.rs

```diff
@@ -101,8 +101,8 @@ fn create_router(conn: Arc<DatabaseConnection>, config: Arc<config::RSCConfig>)
         )
 }
 
-async fn create_standalone_db() -> Result<DatabaseConnection, sea_orm::DbErr> {
-    let db = Database::connect("sqlite::memory:").await?;
+async fn create_standalone_db(db: &str) -> Result<DatabaseConnection, sea_orm::DbErr> {
```

Comment:
It would be nicer to generate unique database names on the fly, this will also require creating those databases however but I think that should be manageable from SeaORM. I think we can get away with just a single database existing that we can connect to, use to make a new database, disconnect, and reconnect to our generated database. If that database name is a UUID it should be unique.